.github/workflows/acceptance-py3.6.yml

```diff
@@ -42,24 +42,24 @@ jobs:
 
       - name: Set python version output
         id: python-version
-        if: steps.changes.outputs.backend == 'true'
+        if: steps.changes.outputs.frontend == 'false'
```

Comment:
It's really unfortunate we can't exit early with a `neutral` status (which I believe would satisfy branch protection checks): see https://github.com/actions/runner/issues/662

This issue has been around for over a year https://github.com/actions/toolkit/issues/146

Reply:
:(